modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -222,46 +220,46 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
                         (bool b) => this[StandaloneCliConfigKeys.RuntimeStateReporting] = b.ToString()},
                     { $"ri|enableroutinginfo=|{StandaloneCliConfigKeys.EnableRoutingInfo}=", "Enable adding routing info to telemetry. By default this is disabled.",
                         (bool b) => this[StandaloneCliConfigKeys.EnableRoutingInfo] = b.ToString() },
-                    { $"{StandaloneCliConfigKeys.UseReversibleEncoding}=", "Use reversible encoding in JSON encoders. Default is false.",
+                    { $"re|reversibleencoding|{StandaloneCliConfigKeys.UseReversibleEncoding}=", "Use reversible encoding in JSON encoders. Default is false.",
                         (bool b) => this[StandaloneCliConfigKeys.UseReversibleEncoding] = b.ToString() },
 
                     // testing purposes
                     { "sc|scaletestcount=", "The number of monitored item clones in scale tests.",
-                        (int i) => this[StandaloneCliConfigKeys.ScaleTestCount] = i.ToString() },
+                        (string i) => this[StandaloneCliConfigKeys.ScaleTestCount] = i.ToString(), true },
```

Comment:
what is the intention with this change? I think this was a numeric, not a bool argument

Reply:
Intention is to hide the setting from the -h output.  the .ToString() before would convert to string, so the new line should remove the .ToString() since it already is a string.  Will fix this up in another PR.